xclim/ensembles/_reduce.py

```diff
@@ -100,7 +100,7 @@ def kkz_reduce_ensemble(
 def kmeans_reduce_ensemble(
     data: xarray.DataArray,
     *,
-    method: dict = None,
+    method: dict,
```

Comment:
Somewhere lower, there's a `if method is None` check with a default value.
Do I understand this change was to fix a typing error? If so, I suggest:
```suggestion
    method: dict | None = None,
```

Reply:
Ah, I was basing this off the numpy docstring. Will revert, thanks!